lib/src/widgets/movie_main_info.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import '../utils/ui_constants.dart';
+
+class MovieMainInfo extends StatelessWidget {
+  const MovieMainInfo({
+    Key? key,
+    required this.movieImage,
+    required this.id,
+    required this.title,
+    required this.releaseDate,
+    required this.voteAverage,
+  }) : super(key: key);
+
+  final String movieImage;
+  final int id;
+  final String title;
+  final String releaseDate;
+  final num voteAverage;
```

Comment:
This can be String, because inside this Widget you use toString() to parse it and that is not necessary here

Reply:
Ok , fixed.